plugins/chat/spec/services/chat/auto_leave_channels_spec.rb

```diff
@@ -26,6 +26,12 @@
           expect { result }.to change { ::Chat::UserChatChannelMembership.count }.from(2).to(0)
         end
 
+        it "only removes memberships for a user when filtering by user_id" do
```

Comment:
This is the spec that's ensuring we're only deleting the membership of that one user.